contracts/Market.sol

```diff
@@ -367,7 +367,8 @@ contract Market is PausableAccessPreset {
   }
 
   /**
-   * @dev Reverts if available stock is being reserved for priority buyers and buyer is not priority.
+   * @dev Reverts if supplier is out of stock or if total available supply in the market is being reserved for priority
+   * buyers and buyer is not priority.
```

Comment:
I think we actually want the original description here because this function only validates priority threshold